src/app/_components/navBar/desktopNavBar.js

```
@@ -1,20 +1,31 @@
-} from "./ui/dropdown-menu";
-import { NAV_BAR_MENU_ITEMS } from "../_consts/nav_bar_menu_items";
+import { useActiveSectionContext } from "../contexts/activeSectionContext";
+import { Button } from "../ui/button";
+import { NAV_BAR_MENU_ITEMS } from "../../_consts/nav_bar_menu_items";
+import { AuthDropdown } from "./authDropdown";
 
 export const NAVBAR_HEIGHT = 64;
 const DROPDOWN_BAR_HEIGHT = 256;
 const MENU_BUTTON_HEIGHT = 20;
 
+if (typeof window !== undefined) {
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal
Unsafe fix: Compare the result of `typeof` with a valid type name


(lint/suspicious/useValidTypeof)